github/GithubApp.ts

```diff
@@ -95,7 +83,10 @@ export class GithubApp extends App implements IPreMessageSentExtend,IPostMessage
         if (await hasGitHubCodeSegmentLink(message)) {
             await handleGitHubCodeSegmentLink(message, read, http, message.sender, message.room, extend);
         }
-        
+        if (await hasGithubPRLink(message)) {
+            await handleGithubPRLinks(message, read, http, message.sender, message.room, extend);
+        }
```

Comment:
@VipinDevelops  do you think adding more of these checks in checkPreMessageSentExtend will hinder performance ?